Corvus.Extensions.Samples/TraversalExtensionsSample.dib

```diff
@@ -0,0 +1,242 @@
+#!markdown
+
+## Traversal Extensions - Sample Notebook
+
+#!markdown
+
+First, add a reference to the `Corvus.Extensions` NuGet package.
+
+#!csharp
+
+#r "nuget: Corvus.Extensions, 1.1.4"
+
+using Corvus.Extensions;
+
+#!markdown
+
+### ForEachAsync()
+
+#!markdown
+
+Execute an async action for each item in the enumerable.
+
+#!csharp
+
+public async Task IsEqualToOne(int x)
+{
+    bool result = x.Equals(1);
+    Console.WriteLine(result);
+}
```

Comment:
If you wrote this in Visual Studio, it would issue a warning because although you've declared this as `async` you don't seem to have used `await`, so this isn't really `async`.

I'd consider adding something like

```cs
await Task.Delay(TimeSpan.FromSeconds(0.5));
```

so that a) it is async, and b) it becomes reasonable clear that `ForEachAsync` waits for the `Task` returned for each item to complete before moving onto the next.

Reply:
Using `await` in all `async` methods now